<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SETTING SCREEN-LCS CHECK WEIGHER SYSTEM</string>
  </property>
  <widget class="QLabel" name="SettingScreen_Image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>settingscreen.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Port">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>155</width>
     <height>155</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Port.png</normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Port.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Product">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>155</width>
     <height>155</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Product.png</normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Product.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Display">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>155</width>
     <height>155</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Display.png</normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Display.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Password">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>155</width>
     <height>155</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Password.png</normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Password.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>156</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Shift">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>155</width>
     <height>155</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Shift.png</normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_Shift.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_backup">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>155</width>
     <height>155</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_backup.png</normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/btn_backup.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>155</width>
     <height>155</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back_home">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/back.png</normaloff>UI SCREEN AND BUTTON FOR CHECKWEIGHER/BUTTON/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
